scripts/backref_cost_attribution_v7_transport.py

```
@@ -0,0 +1,542 @@
+            if digest.get("archive_bytes") != len(archive) or \
+                    digest.get("archive_sha256") != sha256_bytes(archive):
+                raise FrameRefusal("archive builder declaration mismatch")
+            declaration = {**digest, "inventory": inventory,
+                           "chunk_bytes": CHUNK_BYTES,
+                           "chunk_count": (len(archive) + CHUNK_BYTES - 1) // CHUNK_BYTES}
+        success = (operator.returncode == 0 and
+                   operator_status.get("status") == "complete" and
+                   archive_status["status"] == "accepted" and
+                   (lease_state["state"] == "not-attempted" if mode == "rehearse"
+                    else lease_state["state"] == "verified"))
+    except Exception as caught:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)